lib/kubernetes-deploy.rb

```diff
@@ -23,7 +11,34 @@
 require 'active_support/core_ext/hash/slice'
 require 'active_support/core_ext/numeric/time'
 
-class KubernetesDeploy
+module KubernetesDeploy
+  def self.logger=(value)
+    @logger = value
+  end
+
+  def self.logger
+    @logger ||= begin
+      l = Logger.new(STDOUT)
+      l.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
```

Comment:
I'd consider just making this `ENV["LEVEL"]` for full control, with `ENV["DEBUG"]` implying `ENV["LEVEL"] = Logger::DEBUG`. Nitpick though